example/src/example-initial-value.json

```diff
@@ -298,6 +298,27 @@
             }
           ]
         },
+        {
+          "id": 29,
+          "name": "Average spent for meal",
+          "order": 3,
+          "type": "autofield",
+          "required": true,
+          "fn": {
+            "multiline": false,
+            "fnString": "function () { return #8 / #9 || 0 }",
+            "fnColor": {
+              "#8": "#fdbb84",
```

Comment:
the fn color should be defining an answer, not question.
e.g: https://github.com/akvo/akvo-react-form/blob/6802e078de40ee6ed1e17fda961223b58a36dbc3/example/src/rtmis-poc.json#L226

Reply:
Hi mas @dedenbangkit , 
I have pushed the update. Please check when you have time.